package.json

```diff
@@ -58,11 +58,11 @@
   ],
   "scripts": {
     "postinstall": "patch-package",
-    "lint": "lerna run --parallel lint",
+    "lint": "yarn workspaces foreach --all --parallel run lint",
     "watch": "tsc --build ./tsconfig.project.json --watch",
     "build": "tsc --build ./tsconfig.project.json && lerna run build",
-    "test": "lerna run --concurrency 4 test",
-    "clean": "lerna run --parallel clean && lerna clean --yes",
```

Comment:
`lerna clean --yes` should not be needed with modern yarn as it is comes with checksum checks